docs/features/base64-upload-adapter.md

```diff
@@ -0,0 +1,43 @@
+---
+category: features-image-upload
+menu-title: Base64 Upload Adapter
+order: 40
+---
+
+# Base64 Upload Adapter
+
+The {@link module:upload/base64uploadadapter~Base64UploadAdapter} plugin allows inlining images into the editor as a string which is added directly as the editor's content.
+
+## Example
+
+{@snippet features/base64-upload}
+
+## Installation
+
+To add this feature to your editor install the [`@ckeditor/ckeditor5-upload`](https://www.npmjs.com/package/@ckeditor/ckeditor5-upload) package:
```

Comment:
Q: Isn't the ckeditor5-upload package required by other packages and thus present in all editor builds? If so, it's worth leaving a note like:

```
<info-box info>
	The [ckeditor5-upload](https://www.npmjs.com/package/@ckeditor/ckeditor5-upload) package is available by default in all builds. The installation instructions are for developers interested in building their own, custom WYSIWYG editor.
</info-box>
```

Reply:
The `ckeditor5-upload` is required by the `ckeditor5-image`. The `ImageUpload` plugin requires it and the plugin itself is added to our build. I will add your note to the instruction.